images/router/egress/squid.conf

```diff
@@ -0,0 +1,9 @@
+# egress-router squid.conf
+# This is a mostly-transparent HTTP/HTTPS proxy; it only exists to
+# redirect requests, not to authenticate them in any way.
+
+http_port 8080
+http_access allow all
+cache deny all
+access_log none all
+debug_options ALL,0
```

Comment:
"Don't log every request/response"
"Print only critical errors (level 0) in all log domains"

The container ends up logging

    WARNING: Cannot write log file: /var/log/squid/cache.log
    /var/log/squid/cache.log: No such file or directory
             messages will be sent to 'stderr'.

twice at startup. (It shows up in `oc logs` for the pod.) I couldn't find any way to get it to just log to stderr without complaining about it...